software/docker/kafka-can-log/kafka_can_log/isobus.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * linux/can/isobus.h
```

Comment:
Did the kernel ever grow ISOBus support? @awlayton maybe?

Reply:
Yes. J1939 was added to mainline since v5.4. Enabling this will probably require some fun kernel patching.